actionpack/test/controller/render_json_test.rb

```diff
@@ -63,6 +63,13 @@ def render_json_with_extra_options
     def render_json_without_options
       render json: JsonRenderable.new
     end
+
```

Comment:
Maybe we should move this to a different file since this test isn't about rendering json? ...though it does seem to be the only place we really test `render_to_string` 😬

Reply:
I ended up moving the render_to_string into it's own test, leaving this test with json related tests, wonder if that's ok? 🤔